classification2.Rmd

```diff
@@ -527,23 +556,24 @@ vfold_metrics <- workflow() |>
                   add_recipe(cancer_recipe) |>
                   add_model(knn_spec) |>
                   fit_resamples(resamples = cancer_vfold) |>
-                  collect_metrics()
+                  collect_metrics() 
 vfold_metrics
 ```
 
+
 ### Parameter value selection
 
 Using 5- and 10-fold cross-validation, we have estimated that the prediction
 accuracy of our classifier is somewhere around `r round(100*(vfold_metrics %>% filter(.metric == "accuracy"))$mean,0)`%. 
 Whether that is good or not
 depends entirely on the downstream application of the data analysis. In the
-present situation, we are trying to predict a tumour diagnosis, with expensive,
+present situation, we are trying to predict a tumor diagnosis, with expensive,
 damaging chemo/radiation therapy or patient death as potential consequences of
 misprediction. Hence, we might like to 
 do better than `r round(100*(vfold_metrics %>% filter(.metric == "accuracy"))$mean,0)`% for this application.  
 
 In order to improve our classifier, we have one choice of parameter: the number of
-neighbours, $K$. Since cross-validation helps us evaluate the accuracy of our
+neighbors, $K$. Since cross-validation helps us evaluate the accuracy of our
```

Comment:
below here 

"we set the seed prior to tuning to ensure results are reproducible"

same comment as above

maybe here just remove this sentence.